### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jdvr**
First release

### CODE REVIEW

**Review by jnogol — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+![Logo](logo.gif)
```
Comment (jmonterrubio) [inline README.md]: Use full path https://raw.githubusercontent.com/idealista/apache_exporter-role/master/logo.gif

File: README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+## Testing
+
+```
+molecule test
+```
+
+## Built With
+
+![Ansible](https://img.shields.io/badge/ansible-2.2.1.0-green.svg)
```
Comment (jmonterrubio) [inline README.md]: 2.3, same as prerequisites version

Comment (jmonterrubio) [reply README.md]: Also you could add:
Molecule
Testinfra

**Review by dortegau — APPROVED** [review]

**Review by jmonterrubio — CHANGES_REQUESTED** [review]
Take a look to the readme comments

**Review by jmonterrubio — APPROVED** [review]

**Pull request merged**